lib/view_model/send/send_view_model.dart

```diff
@@ -200,19 +206,16 @@ abstract class SendViewModelBase with Store {
       case WalletType.bitcoin:
         final priority = _settingsStore.priority[_wallet.type];
 
-        return bitcoin.createBitcoinTransactionCredentials(
-            outputs, priority);
+        return bitcoin.createBitcoinTransactionCredentials(outputs, priority);
       case WalletType.litecoin:
         final priority = _settingsStore.priority[_wallet.type];
 
-        return bitcoin.createBitcoinTransactionCredentials(
-            outputs, priority);
+        return bitcoin.createBitcoinTransactionCredentials(outputs, priority);
       case WalletType.monero:
         final priority = _settingsStore.priority[_wallet.type];
 
         return monero.createMoneroTransactionCreationCredentials(
-            outputs: outputs,
-            priority: priority);
+            outputs: outputs, priority: priority);
```

Comment:
I think better to disable auto formatter in your editor, because it's hard to see too much changes in PRs which are not related to the task.

Reply:
sorry about that, done.